recipes-tpm/tpm2-tools/tpm2-tools_%.bbappend

```diff
@@ -0,0 +1,24 @@
+LIC_FILES_CHKSUM = "file://doc/LICENSE;md5=a846608d090aa64494c45fc147cc12e3"
+
+SRC_URI = "git://github.com/tpm2-software/tpm2-tools.git;branch=master"
```

Comment:
We will see how stable it will be to use the bbappend files when switching from one yocto release to another. In this case, if need specific dependencies for safeboot, it would be fine to use full recipes (.bb files) here.

Reply:
I've tested this bbapend on zeus and dunfell, both were stable.